### PR TITLE
fixed GenerateXbase #154

### PR DESCRIPTION
**Pull request by cdietrich**
fixed GenerateXbase #154

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XbaseIdeModule.xtend [hunk 1]
```diff
@@ -4,12 +4,10 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- *******************************************************************************/
-package org.eclipse.xtext.xbase.ide;
+ *******************************************************************************/package org.eclipse.xtext.xbase.ide
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XbaseIdeModule.xtend]: linebreak

File: org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XbaseIdeSetup.xtend [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2017 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/package org.eclipse.xtext.xbase.ide
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XbaseIdeSetup.xtend]: linebreak

File: org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XtypeIdeModule.xtend [hunk 3]
```diff
@@ -4,12 +4,10 @@
  * are made available under the terms of the Eclipse Public License v1.0
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
- *******************************************************************************/
-package org.eclipse.xtext.xbase.ide;
+ *******************************************************************************/package org.eclipse.xtext.xbase.ide
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XtypeIdeModule.xtend]: linebreak

File: org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XtypeIdeSetup.xtend [hunk 4]
```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2010-2017 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/package org.eclipse.xtext.xbase.ide
```
Comment (szarnekow) [inline org.eclipse.xtext.xbase.ide/src/org/eclipse/xtext/xbase/ide/XtypeIdeSetup.xtend]: linebreak

**Comment by cdietrich** [comment]
fixed the line breaks

**Pull request merged**